indico/modules/events/registration/client/js/form/fields/DateInput.jsx

```diff
@@ -223,29 +224,32 @@ export function DateSettings() {
     {key: '24h', value: '24h', text: Translate.string('24 hours')},
   ];
   return (
-    <Form.Group widths="equal">
-      <FinalDropdown
-        name="dateFormat"
-        label={Translate.string('Date format')}
-        options={dateOptions}
-        required
-        selection
-        fluid
-      />
-      <Field name="dateFormat" subscription={{value: true}}>
-        {({input: {value: dateFormat}}) => (
-          <FinalDropdown
-            name="timeFormat"
-            label={Translate.string('Time format')}
-            options={timeOptions}
-            placeholder={Translate.string('None')}
-            disabled={!dateFormat.includes('%d')}
-            parse={p.nullIfEmpty}
-            selection
-            fluid
-          />
-        )}
-      </Field>
-    </Form.Group>
+    <div>
```

Comment:
`<>...</>` is better if you just need a single react element there but not an actual div

Reply:
Ok, will do.